contributors/devel/api-conventions.md

```diff
@@ -595,6 +595,16 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### Specifying element ordering for a strategic merge patch on a list [Alpha]
+
+By default, patching a list using the Strategic Merge strategy will provide no guarantees to the final ordering of elements in the patched list.
+
+As of Kubernetes 1.7, Alpha support exists for explicitly specifying the ordering of elements in a during a Strategic Merge Patch.
+This can be done by providing a parallel list containing all of the patchMergeKeys of elements that need to be ordered.
+Any elements missing from the parallel list are guaranteed to appear after elements appearing in the parallel list.
```

Comment:
Or if the server reorders things in the meantime. Intent gets really muddled here. 

Original:
`
A
B
C
D
E
`

how does a client indicate it want to move A to the end, tolerating any server-side reordering of the other elements?